charts/lidarr/README.md

```diff
@@ -1,115 +1,15 @@
-| `Service.loadBalancerIP` | Loadbalance IP for the lidarr GUI | `{}` |
-| `Service.loadBalancerSourceRanges` | List of IP CIDRs allowed access to load balancer (if supported)      | None
-| `ingress.enabled`              | Enables Ingress | `false` |
-| `ingress.annotations`          | Ingress annotations | `{}` |
-| `ingress.labels`               | Custom labels                       | `{}`
-| `ingress.path`                 | Ingress path | `/` |
-| `ingress.hosts`                | Ingress accepted hostnames | `chart-example.local` |
-| `ingress.tls`                  | Ingress TLS configuration | `[]` |
-| `persistence.config.enabled`      | Use persistent volume to store configuration data | `true` |
-| `persistence.config.size`         | Size of persistent volume claim | `1Gi` |
-| `persistence.config.existingClaim`| Use an existing PVC to persist data | `nil` |
-| `persistence.config.storageClass` | Type of persistent volume claim | `-` |
-| `persistence.config.accessMode`  | Persistence access mode | `ReadWriteOnce` |
-| `persistence.config.skipuninstall`  | Do not delete the pvc upon helm uninstall | `false` |
-| `persistence.media.enabled`      | Use persistent volume to store configuration data | `true` |
-| `persistence.media.size`         | Size of persistent volume claim | `10Gi` |
-| `persistence.media.existingClaim`| Use an existing PVC to persist data | `nil` |
-| `persistence.media.storageClass` | Type of persistent volume claim | `-` |
-| `persistence.media.accessMode`  | Persistence access mode | `ReadWriteOnce` |
-| `persistence.media.skipuninstall`  | Do not delete the pvc upon helm uninstall | `false` |
-| `persistence.extraExistingClaimMounts`  | Optionally add multiple existing claims | `[]` |
-| `resources`                | CPU/Memory resource requests/limits | `{}` |
-| `nodeSelector`             | Node labels for pod assignment | `{}` |
-| `tolerations`              | Toleration labels for pod assignment | `[]` |
-| `affinity`                 | Affinity settings for pod assignment | `{}` |
-| `podAnnotations`           | Key-value pairs to add as pod annotations  | `{}` |
-| `deploymentAnnotations`           | Key-value pairs to add as deployment annotations  | `{}` |
-
-Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
-
-```console
-helm install --name my-release \
-  --set timezone="America/New York" \
-    k8s-at-home/lidarr
-```
-
-Alternatively, a YAML file that specifies the values for the above parameters can be provided while installing the chart. For example,
-
-```console
-helm install --name my-release -f values.yaml stable/lidarr
-```
-
----
-**NOTE**
-
-If you get `Error: rendered manifests contain a resource that already exists. Unable to continue with install: existing resource conflict: ...` it may be because you uninstalled the chart with `skipuninstall` enabled, you need to manually delete the pvc or use `existingClaim`.
```

Comment:
The redundant information in a lot of these charts (read: k8s@home charts in general) I would really like to move to a single place (maybe in the top level readme?) so that we only have unique stuff in each chart readme. Thoughts?